### PR TITLE
added: account page

### PR DESCRIPTION
**Pull request by harshalini**
Added account page for vivir ecommerce.
It has login page and signup page.
check it here ->
https://blissful-aryabhata-1df135.netlify.app/login%20page/login.html
Screenshot (308)

### CODE REVIEW

**Review by krishnakant01 — APPROVED** [review]
looks good to merge.

**Comment by krishnakant01** [comment]
UI is really good.

File: cart/cart.html [hunk 1]
```diff
@@ -18,8 +18,8 @@
                     <li><a href="../index.html">Home</a></li>
                     <li><a href="../products.html">Products</a></li>
                     <li><a class="cart-link" href="#">Cart</a></li>
-                    <li><a href="wishlist.html">Wishlist</a></li>
-                    <li><a href="account.html">Account</a></li>
+                    <li><a href="#">Wishlist</a></li>
+                    <li><a href="../login page/login.html">Account</a></li>
```
Comment (tushartiwari7) [inline cart/cart.html]: Try not to include blank spaces while folder naming.

File: login page/login.css [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    color: var(--black);
+}
+#signup-btn:checked ~ label[for="signup-btn"] {
+    border-bottom: solid 5px var(--orange);
+    color: var(--black);
+}
+#login-btn:checked ~ .register-form {
+    display: none;
+}
+#signup-btn:checked ~ .login-form {
```
Comment (tushartiwari7) [inline login page/login.css]: Refrain from using id's too often.

**Review by tushartiwari7 — COMMENTED** [review]
Code is fine. I liked the Naming conventions for colors and box shadow.
Just mentioned some suggestions, review it and you are good to merge.

**Review by tushartiwari7 — APPROVED** [review]

File: cart/cart.html (hunk 1 above)
Comment (harshalini) [reply cart/cart.html]: Yes, will avoid spaces from now on

**Comment by harshalini** [comment]
> UI is really good.

Thanks

File: login page/login.css (hunk 2 above)
Comment (harshalini) [reply login page/login.css]: I tried it with class, but didn't work, that's why used id, but will use class as much as possible, thanks for suggestions.

**Pull request merged**